Modules/vc-module-inventory/VirtoCommerce.InventoryModule.Web/JsonConverters/PolymorphicInventoryJsonConverter.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using VirtoCommerce.InventoryModule.Core.Model;
+using VirtoCommerce.Platform.Core.Common;
+
+namespace VirtoCommerce.InventoryModule.Web.JsonConverters
```

Comment:
Extend converter with all Inventory domain types
Address, FulfillmentCenters and all SearchCriterias